### DISCUSSION TITLE
Broken thumbnails in CP Assets

### DISCUSSION THREAD

**Discussion by dvteam** [statamic/cms#8167] [Troubleshooting]
I have Statamic 4.1.3 installed on my localhost + on shared hosting

Both installations have the same problem with assets - I can't see thumbnails for SOME images there.
![143056](https://github.com/statamic/cms/assets/5386130/847ea4a6-eebf-4f6d-9a77-7c4109e661da)
(No difference under Statamic 4.1.1 and 4.1.2 and 4.1.3)

Say this one is broken after uploading
[CM5A8898](https://github.com/statamic/cms/assets/5386130/29ff4820-5d6d-42fe-9b01-62f882dd009e)

URL which is in HTML is
`http://stiliant.local/cp/thumbnails/cHJvZHVjdHM6OjAxcDJrL2NtNWE4ODk4LmpwZw==/small`
But I redirected to this one which is not exists
`http://stiliant.local/vendor/statamic/cp/svg/filetypes/picture.svg`

If I try to open the file with double-click I can't see any problem, file is there.
I can see the file in cache also. But not in thumbnails in CP.
No any difference if I upload the file into standard Assets container.

Another one is similar but everything is OK
[IMG_6658](https://github.com/statamic/cms/assets/5386130/b5dfe070-b934-4683-95b3-dae1b0a144c3)

What is wrong with the file?

Not sure it is important but statamic is installed into public_html folder and uses rewrite rules in .htaccess to work properly
`RewriteEngine On
Options +FollowSymlinks
RewriteBase /

RewriteCond %{HTTP_HOST} ^.*$
RewriteCond %{REQUEST_URI} !public/
RewriteRule ^(.*)$ public/$1 [L]`

What can I do to fix the issue?

UPD: Tested on Statamic 3.3.68 - it works well with the same image. 
It looks like something wrong with "Image Manipulation" / "Process Source Images" feature. I tried to reduce the image size to make it less than one defined in preset - it works well now with the manually resized image.

**Comment by davideprevosto** [comment]
I had a similar issue. Please double check for the PHP extension `fpassthru`. Is it enabled?

**Reply by dvteam** [reply]
I can't see such extension in list.. But the passthru function was disabled. I removed it from the blacklist and restarted apache server but nothing changed. Image still be broken for already uploaded images and one I just tried to upload also.
